src/researches/findKeywordInPageTitle.js

```diff
@@ -13,10 +14,14 @@ var escapeRegExp = require( "lodash/escapeRegExp" );
  */
 
 module.exports = function( paper ) {
-	var title = paper.getTitle();
-	var keyword = escapeRegExp( paper.getKeyword() );
-	var locale = paper.getLocale();
-	var result = { matches: 0, position: -1 };
+	/*
+	 * NormalizeQuotes also is used in wordMatch, but in order to find the index of the keyword, it's
+	 * necessary to repeat it here.
+	 */
+	const title = normalizeQuotes( paper.getTitle() );
+	const keyword = escapeRegExp( normalizeQuotes( paper.getKeyword() ) );
+	const locale = paper.getLocale();
+	const result = { matches: 0, position: -1 };
 	result.matches = wordMatch( title, keyword, locale );
 	result.position = title.toLocaleLowerCase().indexOf( keyword );
```

Comment:
tl;dr: fixed

Fun fact: apparently this doesn't have anything to do with the changes in this PR, but you stumbled upon a (so far seemingly undiscovered) bug! You happened to use the keyword `Yoast`, which starts with a capital letter. When we match the position of the keyword in the title, we transform the title to lower case, but not the keyword. So the research would try to match `yoast` with `Yoast`, which of course doesn't work. And that must have been the case for God knows how long 😵

Why wasn't this discovered earlier? Well, the position was not matched correctly, but the research correctly discovered that a keyword was present. As you had noticed, this resulted in the weird assessment result `{matches: 1, position: -1}` where you have a match which doesn't have a position. Since this strange combination doesn't fit any of the assessment conditions (because it's nonsensical), the assessment would return no feedback at all.

I've made sure now that the keyword is also set to lower case and I've added a unit test to cover this case.